frontend/src/views/IssueDetail.vue

```diff
@@ -948,8 +976,13 @@ export default defineComponent({
       });
     };
 
-    const selectedTask = computed((): Task | TaskCreate => {
-      return selectedStage.value.taskList[0];
+    const selectedTask = computed((): Task | undefined => {
+      if (state.create) {
```

Comment:
This also needs to be commented as is not obvious why create can't have selected task